browserstack.json

```diff
@@ -0,0 +1,37 @@
+{
+    "browsers": [
+        {
+            "browser": "chrome",
+            "os": "Windows 10",
+            "versions": [
+                "latest"
+            ]
+        },
+        {
+            "browser": "firefox",
+            "os": "Windows 10",
+            "versions": [
+                "latest"
+            ]
+        }
+    ],
+    "run_settings": {
+        "cypress_version": "6.2.0",
+        "cypress_config_file": "./cypress.json",
+        "project_name": "HedgeDoc",
+        "build_name": "build-name",
+        "exclude": [],
+        "parallels": "5",
+        "npm_dependencies": {
+            "typescript": "4.1.3",
+            "cypress-commands": "1.1.0",
+            "cypress-file-upload": "4.1.1"
+        },
+        "package_config_options": {}
+    },
+    "connection_settings": {
+        "local": true,
+        "local_identifier": null
+    },
+    "disable_usage_reporting": false
```

Comment:
Why did you explicitly enable the CLI usage tracking?

Reply:
Copy & paste I guess

Reply:
fixed